### ISSUE TITLE
Does hostobject support array from JavaScript

### ISSUE THREAD

**Issue by yangzhongke** [MicrosoftEdge/WebView2Feedback#1440]
My code in C#
```csharp
[ComVisible(true)]
public class JavaScriptBridge
{
	public int Sum(int[] items)
	{
		return items.Sum();
	}
}
```

My Coe in JavaScript:
```javascript
alert(await bridge.Sum([3, 5, 8]));
```
But it didn't work.
![image](https://user-images.githubusercontent.com/13821363/123023081-6210e280-d409-11eb-8992-dd5856ba4782.png)

I understand that I can work around this issue by using JSON.stringify() to convert an array in JavaScript to a plain string, and pass it to JavaScriptBridge. I just want to know whether there is a more graceful approach?
Thanks.

**Comment by champnic** [comment]
Yes this should be supported - I've added this as a bug on our backlog. Thanks!
